snyk_scm_refresh.py

```diff
@@ -42,11 +42,9 @@ def parse_command_line_args():
 
     return parser.parse_args()
 
-
 def delete_snyk_project(project_id, org_id):
     """Delete a single Snyk project"""
-    snyk_client.organizations.get(org_id).projects.get(project_id).delete()
-
+    return snyk_client.organizations.get(org_id).projects.get(project_id).delete()
```

Comment:
Per our discussion, some error handling here would be helpful. (I'm just commenting so the things don't get lost 👍 )